config/serverless.es.yml

```diff
@@ -77,7 +77,7 @@ xpack.cloud.serverless.project_type: search
 xpack.serverless.search.enabled: true
 
 ## Set the home route
-uiSettings.overrides.defaultRoute: /app/elasticsearch
+uiSettings.overrides.defaultRoute: /app/home
```

Comment:
I'm assuming this was just done for testing purposes and we want to revert before merging?

Reply:
Ouch good catch